.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think as a team we should decide on a uniform style for all our TS code soon. One thing I've done on past teams is  to adopt another company's guide so that personal preferences don't get in the way.

Reply:
should we adopt these prettier config, or should we make the config on `tsconfig.json` that way, regardless of our IDE settings, the tslint will let us know